### PR TITLE
[fasterize/FasterizeEngine#766]: if key == null .. abort action

### PR DESCRIPTION
**Pull request by zimski**

### CODE REVIEW

File: test/measure.js [hunk 1]
```diff
@@ -197,6 +197,19 @@ describe('statsd-profiler', function(){
         });
       });
 
+      describe('with null key' , function () {
+        it('should not call statsd with an Null key ', function (){
+          profiler.timing(null, 300);
+          sinon.assert.neverCalledWith(timing, "html-parser", undefined);
```
Comment (abarre) [inline test/measure.js]: remove the parameter "html-parser" that doesn't make sense for the test.

File: lib/statsd-profiler.js [hunk 2]
```diff
@@ -70,6 +70,10 @@ function measure(options) {
   if (profiler.transformKey) {
     transformKeyArgs.unshift(key);
     key = profiler.transformKey.apply(this, transformKeyArgs);
+    // if key == null then abort
```
Comment (dax) [inline lib/statsd-profiler.js]: Remove this unneeded comment

File: test/measure.js [hunk 3]
```diff
@@ -197,6 +197,20 @@ describe('statsd-profiler', function(){
         });
       });
 
+      describe('with null key' , function () {
+        it('should not call statsd with an Null key ', function (){
+          // make sure that 999999 are only used in this test
+          profiler.timing(null, 999999);
+          sinon.assert.neverCalledWith(timing,sinon.match.any , 999999);
+
+          profiler.count(null, 999999);
```
Comment (dax) [inline test/measure.js]: Create 1 test per case and add a test for `endTime`

**Comment by dax** [comment]
apply minor fixes and it's OK

**Pull request merged**